urth/dashboard/nbexts/bundle_handler.py

```diff
@@ -199,7 +199,7 @@ def _get_local_app(self, abs_nb_path):
 
         :param abs_nb_path:
         '''
-        md = self._create_app_bundle(abs_nb_path, bundle_root=self.nb_dir, overwrite=True, template_fn='local.tpl')
+        md = self._create_app_bundle(abs_nb_path, bundle_root=os.path.join(self.nb_dir, 'local_dashboards'), overwrite=True, template_fn='local.tpl')
```

Comment:
I'm guessing / hoping the code down here makes the directory if it does not exist?